docker-compose.yml

```diff
@@ -10,9 +8,16 @@ app:
     - AWS_ACCESS_KEY
     - AWS_SECRET_KEY
   links:
-    - rabbitmq
-    - datomic
+    - wildfly
 rabbitmq:
   image: rabbitmq:3.5.0-management
-datomic:
-  image: quay.io/democracyworks/datomic-tx:0.9.5153
```

Comment:
Why no datomic? We can run that in a container w/ docker-compose, it's just not containerized in production.